src/be/mcjava/fxml/CustomerScreenController.java

```diff
@@ -0,0 +1,23 @@
+package be.mcjava.fxml;
+
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CustomerScreenController {
+    
+    @FXML
+    private void continuePressed(ActionEvent event) throws Exception{
```

Comment:
the name of this method is very non-specific, i suggest naming it something like continueFromLoginToMenuScreen...
when the method name is clear the code reads a lot more easy